source/ship/cargo_helper.gd

```diff
@@ -0,0 +1,19 @@
+extends Node
+
+func generate_cargo_info(ship: Ship) -> Dictionary:
+	var cargo_info: Dictionary = {}
+	var cargo_items: Array = [
+		"Clothing", "Medical Supplies", "Jewels", "Liquor", "Houseware", "Livestock",
+		"Metals", "Metawheat", "Artwork", "Machinery", "Tools", "Building Material",
+		"Junk", "Slaves", "Small Arms", "Food Products", "Grav Tanks", "Ammunition", "Batteries"
+	]
+	var cargo_holds: int = int(ship.type) + 1
```

Comment:
I think the int conversion is unnecessary here as enums are already ints. But i quess it could be a good stylistic choice.